app/views/articles/_podcast_episodes_feed.html.erb

```diff
@@ -18,12 +18,6 @@
       </div>
```

Comment:
I meant to actually remove the file :D I can't find a place in the codebase where this is used

Reply:
Yeah, the file itself can be deleted :)

Reply:
thanks , deleted